pkg/plugin/validator.go

```diff
@@ -19,19 +19,21 @@ import (
 	"encoding/json"
 	"fmt"
 	"net/http"
-	"net/url"
 	"regexp"
 	"strconv"
-	"strings"
 
 	"github.com/abcxyz/pkg/githubapp"
 	"github.com/google/go-github/v55/github"
 )
 
 const (
-	issueURLPatternRegExp = `^https:\/\/github.com\/([a-zA-Z0-9-]*)\/[a-zA-Z0-9-]*\/issues\/[0-9]+$`
+	regexpNamedGroupKeyOwner   = "owner"
+	regexpNameGroupKeyRepoName = "repoName"
+	regexpNamedGroupKeyIssue   = "issue"
 )
 
+var issueURLPatternRegExp = fmt.Sprintf(`^https:\/\/github.com\/(?P<%s>[a-zA-Z0-9-]*)\/(?P<%s>[a-zA-Z0-9-]*)\/issues\/(?P<%s>[0-9]+$)`, regexpNamedGroupKeyOwner, regexpNameGroupKeyRepoName, regexpNamedGroupKeyIssue)
```

Comment:
Is it intended that the owner and repo can be blank? `*` means "0 or more times"; I think the following string would match:

```text
https://github.com///issues/123
```